src/Components/fe-transaction-review/test/fe-transaction-review.test.js

```diff
@@ -0,0 +1,44 @@
+import { html, fixture, expect, aTimeout } from '@open-wc/testing';
+
+import '../fe-transaction-review.js';
+
+describe('FeTransactionReview', () => {
+  let element;
+  beforeEach(async () => {
+    element = await fixture(
+      html`<fe-transaction-review> </fe-transaction-review>`
+    );
+  });
+
+  it('renders a from label', async () => {
+    element.from = '123456';
```

Comment:
you checked from label is there, but you didnt check whether others label is not dispalyed or not, because we have conditions right ?

Reply:
Will add that test case